modules/govuk_crawler/templates/govuk_sync_mirror.erb

```diff
@@ -41,10 +41,25 @@ function get_error_page() {
   curl -sf https://${STATIC_HOSTNAME}/templates/503.html.erb -o ${MIRROR_ROOT}/error/503.html
 }
 
+function write_timestamps() {
+  website_mirror_dir=$(find "$MIRROR_ROOT" -name "www.*" | head -n 1)
+  homepage_last_modified=$(stat -c '%Y' ${website_mirror_dir}/index.html)
+  upload_time=$(date +%s)
+
+  cat <<EOF > timestamps.txt
+  homepage_last_modified=$homepage_last_modified
```

Comment:
It doesn't look like we're using this value anywhere in the `check_s3_mirror_freshness` script.

Should we just delete lines 46 and 50?

Out freshness checker makes sure that the `timestamps.txt` file is regularly updated, which proves that the mirror script is working, and by definition, that the homepage remains up to date. We don't need to track the homepage age anymore.

Reply:
I agree, will remove those lines 👍